syntax = "proto3";

package syft.lib.syfertext;

message AttributeValues{
    repeated string value = 1;
}